apps/src/applab/sanitizeHtml.js

```diff
@@ -47,10 +47,29 @@ function warnAboutUnsafeHtml(warn, unsafe, safe, warnings) {
     return 0;
   };
 
+  // Do not warn when these attributes are removed.
+  var ignoredAttributes = ['kl_vkbd_parsed'];
```

Comment:
I'm not really sure, but @islemaster  and I had previously hypothesized that they are from some browser plugin. See previous discussion here: https://github.com/code-dot-org/code-dot-org/pull/6895#issuecomment-186941112